### PR TITLE
Pre release r1.1 for kyc-match + kyc-fill-in + CHANGELOG + README

### PR DESCRIPTION
**Pull request by fernandopradocabrillo**
#### What type of PR is this?

Add one of the following kinds:
* documentation
* subproject management

#### What this PR does / why we need it:

Generation of first release-candidate for Know Your Customer API family

This first release candidate r1.1 contains the definition and documentation of the release-candidate of the kyc-match APIO v0.2.0-rc.1 and kyc-fill-in v0.1.1-rc.1

### CODE REVIEW

File: documentation/API_documentation/kyc-fill-in-API-Readiness-Checklist.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 # API Readiness Checklist
 
-Checklist for kyc-fill-in 0.1.1 in FALL24
+Checklist for kyc-fill-in 0.1.1-rc in release r1.1
```
Comment (hdamker) [inline documentation/API_documentation/kyc-fill-in-API-Readiness-Checklist.md:3]: Change according to the selected version number, recommended

```suggestion
Checklist for kyc-fill-in 0.2.0-rc.1 in release r1.1
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,11 +2,74 @@
...
+* for subsequent alpha or release-candidate API versions, the delta with respect to the previous pre-release
+* for a public API version, the consolidated changes since the release of the previous public API version
+
+# r1.1 - rc
+
+## Release Notes
+
+This release contains the definition and documentation of
+* kyc-match v0.2.0-rc.1
+* kyc-fill-in v0.1.1-rc.1
```
Comment (hdamker) [inline CHANGELOG.md:23]:
```suggestion
* kyc-fill-in v0.2.0-rc.1
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,11 +2,74 @@
...
+
+### Fixed
+
+* n/a
+
+### Removed
+
+* n/a
+
+## kyc-fill-in v0.1.1-rc.1
```
Comment (hdamker) [inline CHANGELOG.md:47]:
```suggestion
## kyc-fill-in v0.2.0-rc.1
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,11 +2,74 @@
...
+
+* API definition updated to conform to the Commonalities v0.4.0 and Identity and Consent Management v0.2.0 guidelines included in the CAMARA Fall24 meta-release. No new features were added.
+
+### Added
+
+* n/a
+
+### Changed
+
+* n/a
```
Comment (hdamker) [inline CHANGELOG.md:59]: Here the changed scope would be worth to mention: 
```suggestion
* Scope changed from `know-your-customer:fill-in` to `kyc-fill-in:fill-in`
```

**Review by hdamker — CHANGES_REQUESTED** [review]
Looks good to me, except that the version of kyc-fill-in can't be v0.1.1-rc.1, as there was already a public version with v0.1.1. My recommendation is to use v0.2.0-rc.1 instead as the scope and the base path has been changed in the alignment with Commonalities.

My suggestions assume v0.2.0-rc.1 also for kya-fill-in. If you decide for v0.1.2-rc.1 you need to do the changes yourself.

**Comment by fernandopradocabrillo** [comment]
> Looks good to me, except that the version of kyc-fill-in can't be v0.1.1-rc.1, as there was already a public version with v0.1.1. My recommendation is to use v0.2.0-rc.1 instead as the scope and the base path has been changed in the alignment with Commonalities.
> 
> My suggestions assume v0.2.0-rc.1 also for kya-fill-in. If you decide for v0.1.2-rc.1 you need to do the changes yourself.

Hi @ToshiWakayama-KDDI, I'll update the release PR according to @hdamker comments and go for v0.2.0-rc.

**Comment by sfnuser** [comment]
@fernandopradocabrillo: You may want to update the img references in the yaml specs. Right now they point to Wakayama-san's repo

`KYC_Fill-in_flow`

**Comment by fernandopradocabrillo** [comment]
> @fernandopradocabrillo: You may want to update the img references in the yaml specs. Right now they point to Wakayama-san's repo
> 
> `KYC_Fill-in_flow`

Thanks @sfnuser! I have raised PR #130 to fix this issue. @ToshiWakayama-KDDI whenever you can please take a look and approve so we can merge.

**Review by hdamker — APPROVED** [review]

**Comment by hdamker** [comment]
@ToshiWakayama-KDDI @GillesInnov35 The PR needs one code owner approval to allow @fernandopradocabrillo to merge it and create the release.

**Review by ToshiWakayama-KDDI — APPROVED** [review]
Really sorry for the delay.  
Thank you very much, @fernandopradocabrillo , for creating the PR, and @hdamker, for the reminder.

Best,
Toshi

**Comment by hdamker** [comment]
@ToshiWakayama-KDDI @fernandopradocabrillo Congratulations for the pre-release. The public release for M4 will be much less effort 😄 

I've updated the release tracker pages and closed https://github.com/camaraproject/ReleaseManagement/issues/68.

**Pull request merged**